src/en/controllers-config.md

```diff
@@ -68,8 +78,91 @@ mongo-memory-profile         | string | low      | low/default              | Se
 state-port                   | integer | 37017   |                          | The port to use for mongo connections
 
+### Excluding information from the audit log
+
+See [Audit logging][troubleshooting-logs-audit] for background information on
+this topic.
+
+Excluding information from the audit log is done via the
+`audit-log-exclude-methods` key above, which refers to API calls/methods. The
+recommended approach for configuring the filter is to view the log and make a
+list of those calls deemed undesirable. There is no definitive API call list
+available in this documentation.
+
+The default value of key `audit-log-exclude-methods` is the special value of
+'ReadOnlyMethods'. As the name suggests, this represents all read-only events.
+
+An example value for this key is: `[Pinger.Ping]` where a log message intended
+for removal (by including 'Pinger.Ping' as a value) contains the following
+text:
+
+```no-highlight
+"request-id":4428,"when":"2018-02-12T20:03:45Z","facade":"Pinger","method":"Ping","version":1}}
+```
+
+!!! Important:
+    Only those Conversations whose methods have *all* been excluded will be
+    omitted. For instance, assuming a default filter of 'ReadOnlyMethods', if a
+    Conversation contains several read-only events and a single write event
+    then all these events will appear in the log.
+
+Click the triangle below to reveal a listing of API methods designated by the
+key value of 'ReadOnlyMethods'. 
+
+^# ReadOnlyMethods 
+
+  ```
+  Action.Actions
+  Action.ApplicationsCharmsActions
+  Action.FindActionsByNames
+  Action.FindActionTagsByPrefix
+  Application.GetConstraints
+  ApplicationOffers.ApplicationOffers
+  Backups.Info
+  Client.FullStatus
+  Client.GetModelConstraints
+  Client.StatusHistory
+  Controller.AllModels
+  Controller.ControllerConfig
+  Controller.GetControllerAccess
+  Controller.ModelConfig
+  Controller.ModelStatus
+  MetricsDebug.GetMetrics
+  ModelConfig.ModelGet
+  ModelManager.ModelInfo
+  ModelManager.ModelDefaults
+  Pinger.Ping
+  UserManager.UserInfo
+  Action.ListAll
+  Action.ListPending
+  Action.ListRunning
+  Action.ListComplete
+  ApplicationOffers.ListApplicationOffers
+  Backups.List
+  Block.List
+  Charms.List
+  Controller.ListBlockedModels
+  FirewallRules.ListFirewallRules
+  ImageManager.ListImages
+  ImageMetadata.List
+  KeyManager.ListKeys
+  ModelManager.ListModels
+  ModelManager.ListModelSummaries
+  Payloads.List
+  PayloadsHookContext.List
+  Resources.ListResources
+  ResourcesHookContext.ListResources
+  Spaces.ListSpaces
+  Storage.ListStorageDetails
+  Storage.ListPools
+  Storage.ListVolumes
+  Storage.ListFilesystems
+  Subnets.ListSubnets
+  ```
 
 <!-- LINKS -->
 
 [controllers-creating]: ./controllers-creating.html "Creating a controller"
 [models-config]: ./models-config.html "Configuring models"
+[anchor__excluding-information-from-the-audit-log]: #excluding-information-from-the-audit-log
```

Comment:
I am a big anti-fan of this new pattern you are making up. If you want to differentiate an internal link, why not just use '#'?

Reply:
:ok: